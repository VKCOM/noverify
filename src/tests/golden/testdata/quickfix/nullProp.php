<?php

class Boo {}
class Zoo {}

class Foo {
  /**
   * @var Boo $item
   */
  public $item = null;

  /**
   * @var ?Boo $item1
   */
  public $item1 = null;

  /**
   * @var ?Boo|Zoo $item11
   */
  public $item11 = null;

  /**
   * @var Boo|Zoo $item12
   */
  public $item12 = null;

  /**
   * @var Boo|Zoo $item13
   */
  public $item13=null;

  /**
   * @var Boo|Zoo $item14
   */
  public $item14= null;

  /**
   * @var Boo $item2
   * @var Boo $item3
   */
  public $item2 = null, $item3 = null;

  /**
   * @var Boo $item21
   * @var Boo $item31
   */
  public $item21=null, $item31= null;

  public ?Boo $item5 = null;
  public ?Boo $item6 = null, $item7 = null;

  public ?int $item8 = null;
}
