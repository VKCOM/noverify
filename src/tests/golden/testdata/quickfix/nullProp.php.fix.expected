<?php

class Boo {}
class Zoo {}

class Foo {
  /**
   * @var Boo $item
   */
  public $item;

  /**
   * @var ?Boo $item1
   */
  public $item1 = null;

  /**
   * @var ?Boo|Zoo $item11
   */
  public $item11 = null;

  /**
   * @var Boo|Zoo $item12
   */
  public $item12;

  /**
   * @var Boo|Zoo $item13
   */
  public $item13;

  /**
   * @var Boo|Zoo $item14
   */
  public $item14;

  /**
   * @var Boo $item2
   * @var Boo $item3
   */
  public $item2, $item3;

  /**
   * @var Boo $item21
   * @var Boo $item31
   */
  public $item21, $item31;

  public ?Boo $item5 = null;
  public ?Boo $item6 = null, $item7 = null;

  public ?int $item8 = null;
}
